.pipelines/pr-windows-signed-scripts.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
       -e CLEANUP_ON_EXIT="true" \
       -e CLIENT_ID=${CLIENT_ID} \
       -e CLIENT_SECRET="$(CLIENT_SECRET)" \
-      -e CLUSTER_DEFINITION="examples/e2e-tests/kubernetes/windows/hybrid/definition.json" \
+      -e CLUSTER_DEFINITION="${CLUSTER_DEFINITION}" \

```

Comment:
Where is this defaulted? Maintainers would have to trigger the pipeline manually and set this value?

Reply:
It is defaulted in the AzureDevops pipeline job definition.
Yes, maintainers would need to trigger this manually and override the variable.